src/SoftRobots/component/constraint/model/CableModel.cpp

```diff
@@ -41,12 +40,37 @@ namespace component
 namespace constraintset
 {
 
-using namespace sofa::defaulttype;
+
+using sofa::defaulttype::Vec2Types;
+using sofa::type::Vector2;
+
+template<>
+void CableModel<Vec2Types>::getPositionFromTopology(Coord& position, const int& index)
+{
+    position.x() = m_topology->getPX(index);
+    position.y() = m_topology->getPY(index);
+}
+
+template<>
+SReal CableModel<Vec2Types>::getDistanceToTriangle(Coord position, const Triangle& triangle, sofa::helper::DistancePointTri& proximitySolver, Coord& projectionOnTriangle)
+{
+    Coord p0; Coord p1; Coord p2;
```

Comment:
```suggestion
    Coord p0 { NOINIT };
    Coord p1 { NOINIT };
    Coord p2 { NOINIT };
```
One instruction by line is easier to read.

The `NOINIT` is to avoid the initialization of the vector because you compute it completely just after.